payments-core/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/TransformToPaymentMethodCreateParams.kt

```diff
@@ -1,15 +1,41 @@
-package com.stripe.android.paymentsheet.forms
+package com.stripe.android.paymentsheet.paymentdatacollection
 
+import com.stripe.android.model.PaymentMethodCreateParams
 import com.stripe.android.paymentsheet.elements.IdealBankConfig
-import com.stripe.android.paymentsheet.elements.country.CountryUtils
+import com.stripe.android.paymentsheet.forms.FormFieldValues
 import com.stripe.android.paymentsheet.specifications.SectionFieldSpec.Country
 import com.stripe.android.paymentsheet.specifications.SectionFieldSpec.IdealBank
+import com.stripe.android.view.CountryUtils
 import java.util.Locale
 
 /**
  * This class will transform the fields in a form into a structure as defined by a map.
  */
-class TransformFormToPaymentMethod {
+internal class TransformToPaymentMethodCreateParams {
```

Comment:
does this need to be a class? seems all methods calls are static, we could just define it in a kt file

Reply:
yeah, there are several Transform classes that I have, so I think I would like to make a separate PR to address all of these.